index.html

```diff
@@ -1644,6 +1644,72 @@ <h2>
     </section>
   </section>
   </section>
+
+  <section>
+    <h2>Verification Algorithm</h2>
+    <p>
+This specification might be used with many different key discovery protocols.
+As such, discovery of verification keys is described in a different section of this document, 
+and is assumed to have succeeded prior to beginning the verification process.
+    </p>
+    <p>
+As a general rule, verifiers SHOULD strive to minimize the processing of untrusted data.
+This includes minimizing any processing of the protected header, unprotected header, or payload as part of the key discovery procedures.
+    </p>
+
+    <p>
+When verifying a credential or presentation secured with SD-JWT, the algorithm defined in 
+<a href="https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-06#name-verification-of-the-sd-jwt">
+  Verification of the SD-JWT
+</a> MUST be followed.
+    </p>
+
+    <p>
+When verifying a credential or presentation secured with COSE Sign 1, the algorithm defined in 
+<a data-cite="RFC9052#section-4.4">
+Signing and Verification Process
+</a> MUST be followed.
+    </p>
+
+    <p>
+After verification has succeeded, additional validation checks SHOULD be performed.
```

Comment:
Where do the interdependencies between `iss` and the `issuer` and `holder` properties need to be checked? They seem like [verification](https://www.w3.org/TR/vc-data-model/#dfn-verify) to me, since they establish that there's an unambiguous issuer or presenter.

Reply:
I would call any checking of specific deserialized members "validation" not "verification". 

As a verifier, I might have a policy that allows me to verify credentials from acme, with a known public key regardless of what the string values for "iss" or "issuer" are.

validating the "verified payload" conforms to the core data model, is shared logic between all securing formats... and I prefer to cite it from this document, not duplicate it.